fix: correct grammatical error (#134)
smartmind-team/thanosql-docs: docs/en/getting_started/hello_ThanoSQL.md

```
[![IMAGE](/img/thanosql_api/restapi_token_img2.jpg)](/img/thanosql_api/restapi_token_img2.jpg) 

> !!! notice "How to create a query statement in the ThanoSQL Workspace" 

Reviewer: ```suggestion
!!! notice "How to create a query statement in the ThanoSQL workspace" 

```

>     You can use either a one-line or multi-line format when writting ThanoSQL queries.

Reviewer: ```suggestion
    You can use either a one-line or multi-line format when writing ThanoSQL queries.

```

>     - One-line queries are primarily used when returning queries in table form and assigning tables to variables. It is also used to assign ThanoSQL cell magic and Token, as shown in step 1 and 2.

Reviewer: ```suggestion
    - One-line queries are primarily used when returning queries in table form and assigning tables to variables. It is also used to assign ThanoSQL cell magic and a token, as shown in steps 1 and 2.

```
